backend/FacilityBooking/FacilitiesAPI.py

```diff
@@ -264,19 +272,11 @@ def add_booking():
             {
                 "facilityID": formData.get("facilityID"),
                 "$or": condition
-            },
-            {"_id": 0}
-        ).count()
```

Comment:
You did not retrieve the list by wrapping the result in list(). _id also needs to be removed before returning.

Please test your code